streaming/stream_session.cc

```diff
@@ -157,6 +157,8 @@ void stream_manager::init_messaging_service_handler(abort_source& as) {
                                 break;
                             case stream_mutation_fragments_cmd::error:
                                 return make_exception_future<mutation_fragment_opt>(std::runtime_error("Sender failed"));
+                            case stream_mutation_fragments_cmd::abort:
+                                return make_exception_future<mutation_fragment_opt>(rpc::canceled_error());
```

Comment:
Is this something that works out-of-the-box in mixed clusters?

Reply:
Now it is confirmed, it works in mixed cluster